esp/scm/ws_workunits.ecm

```diff
@@ -2256,7 +2256,7 @@ ESPservice [
     ESPmethod [cache_seconds(60), min_ver("1.61")] WUGetNumFileToCopy(WUGetNumFileToCopyRequest, WUGetNumFileToCopyResponse);
     ESPmethod [min_ver("1.71")] WUDetails(WUDetailsRequest, WUDetailsResponse);
```

Comment:
No caching on this?  The current approach seems like you are very likely to miss the attribute off new entries.

Reply:
"miss the attribute off new entries".  We could have a service level attribute that meant "for this service cache_senconds" must be defined for every method.  That way the developer has to think about what it should be, including the case where it should be "cache_seconds(0)".  That's possibly better than defaulting either way and having the behavior negatively affected.